### DISCUSSION TITLE
Hiding mouse cursor when navigating and typing

### DISCUSSION THREAD

**Discussion by jordanthornquest** [zed-industries/zed#7466] [Feature Requests]
When using the keyboard to navigate and type in native macOS applications, the mouse cursor typically disappears. It appears this behavior doesn't occur in Zed. Is this expected? Can this be toggled? Particularly when using Vim Mode, this would be super slick!

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
